docs/reference/esql/functions/kibana/docs/mv_percentile.md

```diff
@@ -0,0 +1,11 @@
+<!--
+This is generated by ESQL's AbstractFunctionTestCase. Do no edit it. See ../README.md for how to regenerate it.
+-->
+
+### MV_PERCENTILE
+Converts a multivalued field into a single valued field containing the value at which a certain percentage of observed values occur.
```

Comment:
Something that I picked up from code: `percentile` that's negative or greater than 100 will return `null`. Should this be mentioned in docs?

Reply:
Looks like a very good thing to add to the field description 👀
Adding it

Reply:
Thinking out loud here... if we default to `null` because some kind of validation of the `percentile` value failed, shouldn't we also issue a warning header for this? Of course, if we don't error out if the value is wrong.

Just looked at `top` and we throw a validation exception if `desc` or `asc` is not provided. Actually, `mv_sort` does this as well but those are constants.

I think we should be consistent here I _think_ adding a warning to the list of warning headers.

Reply:
Oh, absolutely. Adding it

Reply:
Thank you for making the change.

Just asking for more opinions on the approach here:  @bpintea @nik9000 whyt about the validation of mv_percentile `percentile` value? Right now it's creating a `null` if the value is outside of 0-100 interval and a warning in headers. `top` for example, for `top(salary, -1, "asc")` is issuing an error about the validity of the negative value of the second argument. Thanks.

Reply:
If we can catch the thing at query compile time I'd love to make an error. But at runtime a warning and `null` is probably the best we're going to do.

Reply:
We can add that check with the Validatable interface, after https://github.com/elastic/elasticsearch/issues/100634 is done. The idea is to also move the TOP validation there I think, as validation is executed after constant propagation